test/Microsoft.AspNetCore.Server.Kestrel.Core.Tests/FrameResponseHeadersTests.cs

```diff
@@ -59,14 +64,14 @@ public void InitialDictionaryIsEmpty()
         [InlineData("Ser\u0080ver", "Data")]
         [InlineData("Server", "Da\u0080ta")]
         [InlineData("Unknown\u0080-Header", "Data")]
-        [InlineData("Ser�ver", "Data")]
-        [InlineData("Server", "Da�ta")]
-        [InlineData("Unknown�-Header", "Data")]
-        [InlineData("Ser�ver", "Data")]
-        [InlineData("�erver", "Data")]
-        [InlineData("Server", "Da�ta")]
-        [InlineData("Unknown�-Header", "Data")]
-        [InlineData("Ser�ver", "Data")]
+        [InlineData("Ser™ver", "Data")]
+        [InlineData("Server", "Da™ta")]
+        [InlineData("Unknown™-Header", "Data")]
+        [InlineData("Ser™ver", "Data")]
+        [InlineData("šerver", "Data")]
+        [InlineData("Server", "Dašta")]
+        [InlineData("Unknownš-Header", "Data")]
+        [InlineData("Seršver", "Data")]
```

Comment:
If you save wrong it it can easily change; in coreclr they force use of surrogate pairs \uxxx to avoid the risk (VS seems to like using non extended encodings) - which made me sad as I had originally used a bunch of emojis 😞 

However it should stand out on a PR; so is probably ok?